### PR TITLE
feat: easy share to twitter

### PR DESCRIPTION
**Pull request by codingki**

### CODE REVIEW

**Comment by pveyes** [comment]
hi @codingki can you merge/rebase to latest master, just added prettier so this change doesn't look so big thank you 🙇

**Comment by codingki** [comment]
@pveyes sure

**Comment by codingki** [comment]
@pveyes done!

File: components/StatsModal.tsx [hunk 1]
```diff
@@ -63,6 +63,38 @@ export default function StatsModal(props: Props) {
...
+        } else if (answer.includes(char)) {
+          text += "🟨";
+        } else {
+          text += "⬛";
+        }
+      });
+      text += "%0A";
+    });
+
+    text += "%0Ahttps://katla.vercel.app";
```
Comment (pveyes) [inline components/StatsModal.tsx:93]: can we reuse this function instead and use encode URI for twitter share?

File: components/StatsModal.tsx [hunk 2]
```diff
@@ -64,34 +68,9 @@ export default function StatsModal(props: Props) {
...
-          text += "⬛";
-        }
-      });
-      text += "%0A";
-    });
-
-    text += "%0Ahttps://katla.vercel.app";
-    const shareToTwitter = `https://twitter.com/intent/tweet?text=${text}`;
+    const text = generateText()
+    const encodeURI = text.replaceAll("\n", "%0A")
```
Comment (codingki) [inline components/StatsModal.tsx:72]: wait, does javascript have a function to encode to URI

Comment (pveyes) [reply components/StatsModal.tsx:72]: yes, using `encodeURI` or `encodeURIComponent` I forget which one does what lol

Comment (codingki) [reply components/StatsModal.tsx:72]: but `\n` will become `%5Cn`

**Review by pveyes — APPROVED** [review]
I don't have any strong opinion on this, whatever works 🙏

**Comment by pveyes** [comment]
Thanks for making this PR @codingki 🥳

**Pull request merged**